README.md

```diff
@@ -831,7 +831,7 @@ The differences being that as a result of defining it as a property, it is now l
 
 ### `emit (elem, eventName, eventOptions = {})`
 
-Emits a `CustomEvent` on `elem` that `bubbles` and is `cancelable` by default. This is useful for use in components that are children of a parent component and need to communicate changes to the parent.
+Emits a `CustomEvent` on `elem` that `composed`, `bubbles` and is `cancelable` by default. This is useful for use in components that are children of a parent component and need to communicate changes to the parent.
```

Comment:
Maybe just "an event" as the actual instance type did change, but it's not considered part of our public API.